plugins/admission/pkg/admission/admission.go

```diff
@@ -19,11 +19,14 @@ import (
 var organization string
 var hostname string
 var cluster string
+var version string
 
 func init() {
 	organization = os.Getenv("FAIRWINDS_ORGANIZATION")
 	hostname = os.Getenv("FAIRWINDS_HOSTNAME")
 	cluster = os.Getenv("FAIRWINDS_CLUSTER")
+	version = "1.0.0"
```

Comment:
can we just get this from `version.txt` using https://pkg.go.dev/embed? You might need to add it to the dockerfile.